apps/amo/tests/test_csp_headers.py

```diff
@@ -0,0 +1,95 @@
+from django.conf import settings
+
+import lib.settings_base as base_settings
+
+import amo.tests
+import mock
+
+
+@mock.patch.object(settings, 'CSP_REPORT_ONLY', False)
```

Comment:
You can use [django.test.utils.override_settings](https://docs.djangoproject.com/en/1.9/topics/testing/tools/#django.test.override_settings) instead. Do we need that for the csp to be added to the response?